assets/js/vue-apps/form-components/address-lookup.vue

```diff
@@ -52,8 +52,11 @@ export default {
                     'Apply: AFA: Contacts: User needs address history'
                 ]);
             }
+
             if (value === 'yes') {
                 this.currentState = states.NotRequired;
+            } else if (value === 'unknown') {
```

Comment:
Not a big fan of this, could we not rely on the value being undefined rather than introducting a false value?

Would something like this work?

```js
if (value === undefined || value === 'yes') {
 this.currentState = states.NotRequired;
}
```

Reply:
👍 – was even simpler as `!value` does what we need.